src/app/app.component.ts

```diff
@@ -2,7 +2,6 @@ import { Component } from '@angular/core'
 
 @Component({
   selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css'],
+  templateUrl: './app.component.html'
```

Comment:
why did you remove styles link of the component?

Reply:
because we dont need components styles, we will use the global styles in app

Reply:
no, add them, each file will have it's own styles, even if they are empty